### PR TITLE
Resolves issue #1833

### PR DESCRIPTION
**Pull request by Rember**
The welder item no longer displays a "cannot use" message after waiting out the usage times after being used in an unusable state. For example:

Before: The player tries to use the welder on a weldable item while the welder is turned off, the player uses the turned off welder and waits out the timer bar thing, the player recieves a message that the welder cannot be used after waiting out the timer bar thing.

Now: The player tries to use the welder on a weldable item while the welder is turned off, the player uses the turned off welder and instantly recieves the "cannot use" message, and doesn't have to wait.

### CODE REVIEW

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs [hunk 1]
```diff
@@ -123,21 +128,15 @@ bool ExtraCheck()
...
         private bool TryWeld(float value, IEntity? user = null, bool silent = false)
         {
-            if (!WelderLit)
-            {
-                if(!silent) _notifyManager.PopupMessage(Owner, user, Loc.GetString("The welder is turned off!"));
-                return false;
-            }
-
-            if (!CanWeld(value))
+            if (!CanWeld(value, user))
```
Comment (RemieRichards) [inline Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:139]: This if is suspect now.

imagine if you use a welder that's turned off here, you'll get both messages:

"The welder is turned off!"
"The welder does not have enough fuel for that!"

Comment (metalgearsloth) [reply Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:139]: When transferring from fuel tank for example it says it's off and that it has also transferred.

IMO I'd leave the notification out of CanWeld and just let the caller handle it which should fix it.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
The other UseTool method will also need a !CanWeld check which will stop the DoAfter showing up as well.

**Comment by Rember** [comment]
I kinda messed this up on my end, gonna close this one for now

**Pull request closed without merging**